examples/tags_example_test.go

```diff
@@ -44,11 +44,11 @@ func demonstrateTagsCRUD() error {
 	t.TCPFlags = stringPointer("7")
 	t.Protocol = stringPointer("6,17")
 	t.ASN = stringPointer("101,102,103")
-	t.LasthopAsName = stringPointer("as1,as2,as3")
-	t.NexthopAsn = stringPointer("51,52,53")
-	t.NexthopAsName = stringPointer("as51,as52,as53")
-	t.Nexthop = stringPointer("192.168.7.1,192.168.7.2")
-	t.BGPAspath = stringPointer("201,202,203")
+	t.LastHopAsName = stringPointer("as1,as2,as3")
+	t.NextHopAsn = stringPointer("51,52,53")
```

Comment:
ASN could be all caps, because it is an acronym (Autonomous System Number).

Reply:
Done